.github/workflows/schedule-stale.yml

```diff
@@ -36,10 +35,6 @@ jobs:
             Thanks again for being part of the Gatsby community! 💪💜
           EXEMPT_ISSUE_LABELS: |
             not stale
-            important
```

Comment:
Do we really want to drop `important` and `no triage`? Especially the important one seems like we wouldn't just want to let it go stale?

Reply:
@pieh reasoning (and I agree) is that we should first mimic the current behavior and later could add other labels. Right now it's explicit ("not stale") on purpose, adding other ones would make things a bit more opaque.

Reply:
I think that stale bot should be optional additional layer over regular issue triaging. If something is `important` it can get stale as well and might make sense for stale bot to close it (we do need to monitor what issues get stale warnings and get closed - that's what slack notification is for). I can see argument for `no triage` not closing the issue, but as @LekoArts mention this PR hopes for 1-1 migration and we can "mess" with config in follow up